symphony/bdk/core/config/model/bdk_config.py

```diff
@@ -0,0 +1,55 @@
+from symphony.bdk.core.config.model.bdk_bot_config import BdkBotConfig
+from symphony.bdk.core.config.model.bdk_server_config import BdkServerConfig
+from symphony.bdk.core.config.model.bdk_client_config import BdkClientConfig
+from symphony.bdk.core.config.model.bdk_ssl_config import BdkSslConfig
+
+
+class BdkConfig(BdkServerConfig):
+    """Class containing the Bdk configuration """
+
+    def __init__(self, **kwargs):
+        super().__init__(scheme=kwargs.get("scheme"), host=kwargs.get("host"), port=kwargs.get("port"),
+                         context=kwargs.get("context"))
+        self.__agent = BdkClientConfig(self, kwargs.get("agent"))
+        self.__pod = BdkClientConfig(self, kwargs.get("pod"))
+        self.__key_manager = BdkClientConfig(self, kwargs.get("keyManager"))
+        self.__session_auth = BdkClientConfig(self, kwargs.get("sessionAuth"))
+
+        self.__bot = BdkBotConfig(kwargs.get("bot"))
+        self.__ssl = BdkSslConfig(kwargs.get("ssl"))
+
+    @property
+    def agent(self): return self.__agent
```

Comment:
If we don't need any specific logic in getters and setters, IMHO we can use fields without underscores and remove the properties